src/main/java/uk/gov/cslearning/record/domain/Invite.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.cslearning.record.domain;
+
+import javax.persistence.*;
+
+@Entity
+public class Invite {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    @OneToOne(cascade = CascadeType.ALL)
+    private Event event;
+
+    @Column(nullable = false)
```

Comment:
Shouldn't we reflect restrictions in the Entities? IE `length=50`

`learner_email` VARCHAR(50) NOT NULL,`